connectors/protocol/connectors.py

```diff
@@ -806,7 +828,19 @@ async def sync_done(self, job, cursor=None):
             doc["last_indexed_document_count"] = job.indexed_document_count
             doc["last_deleted_document_count"] = job.deleted_document_count
 
-        await self.index.update(doc_id=self.id, doc=doc)
+        if self.index.feature_use_connectors_api:
+            await self.index.api.connector_update_status(
+                connector_id=self.id, status=connector_status.value
+            )
+            await self.index.api.connector_update_error(
+                connector_id=self.id, error=job_error
+            )
+            await self.index.api.connector_update_last_sync_info(
+                connector_id=self.id, last_sync_info=last_sync_information
+            )
```

Comment:
And same thing here - is there any chance we unite these three into one?

Reply:
See answer above about other 3 calls